api/v1beta1/preflightvalidation_types.go

```diff
@@ -117,6 +117,9 @@ func (p *PreflightValidation) ConvertFrom(srcRaw conversion.Hub) error {
 		for _, v := range src.Status.Modules {
 			v := v
 			p.Status.CRStatuses[v.Namespace+"/"+v.Name] = &v.CRBaseStatus
+
```

Comment:
lets just add a TODO to remove it as soon as we stop supporting < 2.1

Reply:
I think we should keep this code until we deprecate `v1beta1` entirely.